_scripts/go-vitess/Makefile

```diff
@@ -1,56 +1,41 @@
-# Tooling to create the package `gopkg.in/src-d/go-vitess.v0`.
+# Tooling to create the package `gopkg.in/src-d/go-vitess.v1`.
```

Comment:
I would prefer to have this script separated in another PR as you did: https://github.com/src-d/go-mysql-server/pull/384

Reply:
yeah, it's already there 
https://github.com/src-d/go-mysql-server/pull/384. I just didn't revert changes on this branch (so doesn't matter in what order we'll merge it, hopefully there will not be conflicts).

Reply:
#384 is already merged. This file shouldn't be here now.